guides/common/modules/proc_running-installer.adoc

```diff
@@ -3,17 +3,62 @@
 [id="running-the-{project-context}-installer_{context}"]
 = Running the {Project} installer
 
-The Foreman installer is a collection of Puppet modules that installs {Project} based on native operating system packages.
-
-The installation run is non-interactive by default.
-The script displays its progress and writes logs to `{installer-log-file}`.
+Install the required packages and run the {Project} installer utility.
 
 .Procedure
-* Run the {Project} installer:
+ifdef::foremanctl[]
+[tabs]
+====
+Puppet-based installation::
+endif::[]
+. Upgrade all packages:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-upgrade}
+----
+. Install {project-installer-package}:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-install} {project-installer-package}
+----
+. Run the {Project} installer:
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
 # {installer-scenario}
 ----
 +
+The {foreman-installer} installer is a collection of Puppet modules that installs {Project} based on native operating system packages.
++
+The installation run is non-interactive by default.
+The script displays its progress and writes logs to `{installer-log-file}`.
++
 For a list of the available customization options, run `{foreman-installer} --help` or `{foreman-installer} --full-help`.
+ifdef::foremanctl[]
```

Comment:
If the `:foremanctl:` attribute will be defined for all builds, then this expression will always be true. Is that intentional?